cap/cps.py

```diff
@@ -0,0 +1,114 @@
+def check_cps(data,
+              max_cps: float = 15,
+              diviation: float = 1.5):
+    """
+    Checks if the caption group follows the rule of around 15 characters per
+    second.
+
+    Args:
+        data: Caption group according to our custom Caption-list datastructure.
+        max_cps: Indicates the optimal characters per second, which is 15.
+        diviation: Indicates how much it can diviate from the optimal 15
+            characters per second.
+
+    Returns:
+        1 if the characters per second is too high.
+        0 if the characters per second is just right.
+        -1 if the characters per second is too low.
+    """
+    tot_time = data[-1].end - data[0].start
+    characters = len(' '.join([word.text for word in data]))
+    cps = characters / tot_time
+    print(cps)
+
+    if cps > max_cps + diviation:
+        return 1
+
+    elif cps < max_cps - diviation:
+        return -1
+
+    else:
+        return 0
```

Comment:
```suggestion
    if cps > max_cps + diviation:
        return 1

    if cps < max_cps - diviation:
        return -1

    return 0
```

btw het is ook deviation ipv diviation